src/components/NcActions/NcActions.vue

```diff
@@ -829,7 +829,9 @@ export default {
 			if (focusElement) {
 				this.removeCurrentActive()
 				const liMenuParent = focusElement.closest('li.action')
-				focusElement.focus()
+				setTimeout(() => {
+					focusElement.focus()
+				}, 10)
```

Comment:
After change the state, it is not affect the view (DOM) immediately.
To "wait" for it, just use [`this.nextTick(() => {})`](https://v2.vuejs.org/v2/api/#vm-nextTick)

```
this.$nextTick(() => focusElement.focus())
```

Reply:
I know, and I tried this. But it does not work. Even with multiple nextTick.

Reply:
😨